README.md

```diff
@@ -85,12 +86,25 @@ subscribeEvent_ :: forall m a. MonadEffect m => MonadCleanup m => (a -> Effect U
 #### [Callback](https://pursuit.purescript.org/packages/purescript-specular/docs/Specular.Callback#t:Callback)
 
 `Callback a` represents an action handler which consumes a value of type `a`. Think of it as `a -> Effect Unit`.
+This represenation is likely to change in the future to `a -> Effect Unit`.
 
 ```purescript
 -- | Trigger the action in Effect.
 triggerCallback :: forall a. Callback a -> a -> Effect Unit
 ```
 
+The DOM API that Specular exposes accepts callbacks, please also have a look at `Ref` bellow.
+
+```purescript
+-- | Create a button that triggers `removeTask` callback
+el "button" [class_ "close", attrs ("type":="button" ), onClick_ $ removeTask task ] do
```

Comment:
More idiomatic:

```suggestion
el "button" [class_ "close", attr "type" "button", onClick_ $ removeTask task ] do
```

`Attrs` are rarely used in the new API, individual `Prop`s are preferred because then converting one of them to the dynamic version is easier

Reply:
Makes sense, there are several other examples here that use `attrs`  instead of `attr`.